docs/source/libraries.rst

```diff
@@ -9,53 +9,53 @@ Supported Peripherals
-+---------------+---------------+---------------+---------------+-------------------------------+
-| Pulse Counter | No            | No            | No            |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| RMT           | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| SDIO          | No            | No            | No            |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| SPI           | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| Timer         | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| Temp. Sensor  | Not Supported | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| Touch         | Yes           | Yes           | Not Supported |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| TWAI          | No            | No            | No            |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| UART          | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| USB           | Not Supported | Yes           | Yes           | ESP32-C3 only CDC/JTAG        |
-+---------------+---------------+---------------+---------------+-------------------------------+
-| Wi-Fi         | Yes           | Yes           | Yes           |                               |
-+---------------+---------------+---------------+---------------+-------------------------------+
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Peripheral    | ESP32         | ESP32-S2      | ESP32-C3      | ESP32-S3      | Comments               |
++===============+===============+===============+===============+===============+========================+
+| ADC           | Yes           | Yes           | Yes           | Yes           |                        |
++---------------+---------------+---------------+---------------+---------------+------------------------+
+| Bluetooth     | Yes           | Not Supported | Not Supported | Not Supported | Bluetooth Classic      |
```

Comment:
ESP32-S3 supports BLE, the same for ESP32-C3
We can replace Yes and Not Supported by "BLE and Classic" or "BLE only".
Remove the comments - "Classic"